### ISSUE TITLE
Cannot find module '@strapi/plugin-upload/server/utils/file'

### ISSUE THREAD

**Issue by 7alip** [nicolashmln/strapi-plugin-responsive-image#27]
It seems `@strapi/plugin-upload/server/utils/file` doesn't exist in v4.6.1

https://github.com/strapi/strapi/tree/main/packages/core/upload/server/utils

```bash
[2023-02-17 03:49:29.270] error: Cannot find module '@strapi/plugin-upload/server/utils/file'
Require stack:
- /Users/7alip/code/wsvvrijheid/wsvvrijheid-monorepo/apps/api/node_modules/strapi-plugin-responsive-image/server/services/image-manipulation.js
- /Users/7alip/code/wsvvrijheid/wsvvrijheid-monorepo/apps/api/dist/src/extensions/upload/strapi-server.js
- /Users/7alip/code/wsvvrijheid/wsvvrijheid-monorepo/apps/api/node_modules/@strapi/utils/lib/import-default.js
- /Users/7alip/code/wsvvrijheid/wsvvrijheid-monorepo/apps/api/node_modules/@strapi/utils/lib/index.js
- /Users/7alip/code/wsvvrijheid/wsvvrijheid-monorepo/apps/api/node_modules/@strapi/strapi/lib/Strapi.js
- /Users/7alip/code/wsvvrijheid/wsvvrijheid-monorepo/apps/api/node_modules/@strapi/strapi/lib/index.js
- /Users/7alip/code/wsvvrijheid/wsvvrijheid-monorepo/apps/api/node_modules/@strapi/strapi/lib/commands/transfer/utils.js
- /Users/7alip/code/wsvvrijheid/wsvvrijheid-monorepo/apps/api/node_modules/@strapi/strapi/bin/strapi.js
```

**Comment by 7alip** [comment]
Fixed #26

**Comment by thexpand** [comment]
Well, fixed, but not merged yet. I had to use `patch-package` to fix this in production until I wait for the PR to be merged.
